lib.php

```diff
@@ -21,6 +21,10 @@
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
+defined('MOODLE_INTERNAL') || die();
+
 function tool_heartbeat_status_checks() {
     return [new \tool_heartbeat\check\authcheck()];
 }
+
+tool_heartbeat\logger::register_shutdown_handler();
```

Comment:
I don't think lib.php is always guaranteed to be loaded, I think this needs to be inside a callback such tool_heartbeat_after_config but this only landed in 3.8. We might not need older support, if we do there are other callbacks we can use

https://tracker.moodle.org/browse/MDL-66340